code/modules/admin/secrets.dm

```diff
@@ -379,6 +380,31 @@
 			for(var/obj/machinery/light/L in machines)
 				L.broken()
 
+		if("anime")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","CC")
+			message_admins("[key_name_admin(usr)] made everything kawaii.")
+			for(var/mob/living/carbon/human/H in mob_list)
+				H << sound('sound/AI/animes.ogg')
+
+				if(H.dna.species.id == "human")
```

Comment:
Plasmamen don't even have tail_human or ears included in their mutant_bodyparts list, you can set the variables but the parts won't show up until they change to a species that does have them (right now only humans). It's similar to how you can set a value for snout or frills on a human but it won't show up until the human changes into a lizard. The renderer checks if a mutant body part is in the species' mutant_bodyparts list before it even tries to render it.